docs/source/conf.py

```diff
@@ -21,6 +21,21 @@
 sys.path.insert(0, os.path.abspath("../../"))
 
 
+def copy_notebooks_into_docs_folder(app):
+    # .ipynb files must be inside the docs/ folder for Jupyter to be able to convert them
+    currpath = os.path.dirname(os.path.realpath(__file__))
+    source = os.path.join(currpath, "../../examples/notebooks")
+    destination = os.path.join(currpath, "notebooks")
+    print(
+        f"current path: {currpath}; source path: {source}; destination path: {destination}"
+    )
+    try:
+        shutil.rmtree(destination)
```

Comment:
Do we need this `rmtree`? It seems potentially dangerous, most of the time we would execute on the remote machine but I could see a case where we would execute locally and may get bad surprise.

Reply:
Good point, I updated it to `shutil.copytree(source, destination, dirs_exist_ok=True)`, the `dirs_exist_ok=True` is what I was missing previously. 

https://docs.python.org/3/whatsnew/3.8.html#shutil